xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/ledger/LedgerHeaderJsonTests.java

```diff
@@ -77,7 +78,7 @@ public void deserializeLedgerHeaderWithTransactions() throws JsonProcessingExcep
         .build();
 
     String json = "{\n" +
-        "  \"ledger_index\" : \"54300940\",\n" +
```

Comment:
I think this needs to serialize as a String, because can't a `LedgerIndex` occasionally take a [string form](https://xrpl.org/xrp-api-data-types-ledgerindex.html#ledgerindex)?

e.g., couldn't this be `validated`?

Reply:
This is just the JSON that the client sends right? In which case Jackson doesn't require (but also allows) quotes around the number.

Reply:
We should add a test though that verifies serialization using `"ledger_index": "validated"`

Reply:
2 things here:
1) `LedgerHeader`s always come from rippled responses, and `LedgerIndex`es in responses are always numbers, not `String`s, which deserialize fine because of our custom deserializer.
2) For some reason `LedgerHeader.ledgerIndex` comes back from rippled as a numerical `String`, not just a number. This is fine because of point (1), but these JSON tests were failing because `LedgerIndex` gets serialized to a number, so I just fixed the JSON

Reply:
I sync'd with @nkramer44 and he told me why this is all correct, and I concur. He's going to write a unit test for `"ledger_index": "validated"` as a final verification.

Reply:
@nhartner I think we should unit test `LedgerIndexSerializer` and `LedgerIndexDeserializer`, but I don't think unit testing `"ledger_index": "validated"` makes sense for result objects, since `ledger_index` will always be a number (or a numerical `String`) in response objects. How's that sound?

Reply:
@nhartner Just added tests. Lmk what you think